edit/style.go

```diff
@@ -7,19 +7,19 @@ var (
 	//styleForPrompt           = ""
 	//styleForRPrompt          = "inverse"
 	styleForCompleted        = styles{"dim"}
+	styleForCompletedHistory = styles{"dim"}
 	styleForMode             = styles{"bold", "lightgray", "magenta"}
 	styleForTip              = styles{}
-	styleForCompletedHistory = styles{"dim"}
 	styleForFilter           = styles{"underlined"}
 	styleForSelected         = styles{"inverse"}
 	styleForScrollBarArea    = styles{"magenta"}
 	styleForScrollBarThumb   = styles{"magenta", "inverse"}
 	styleForSideArrow        = styles{"inverse"}
 
 	// Use black text on white for completion listing.
-	styleForCompletion = styles{"black", "bg_white"}
+	styleForCompletion = styles{}
 	// Use white text on black for selected completion.
-	styleForSelectedCompletion = "inverse"
+	styleForSelectedCompletion = styles{"black", "bg-white", "bold"}
```

Comment:
I see your point, even though I'd argue that 99% of terminals still have a black background - even if OSX defaults to black on white, personally I haven't seen anyone managing to cope with that for more than 5 minutes - especially power users who dare to switch their shell.

I'd suggest also setting a sane default for 'styleForCompletion' then. My initial decision to get rid of inversion was indeed to due the clash with ls colors.